docs/pro/pricing.md

```diff
@@ -0,0 +1,274 @@
+
+The Pro Pricing feature is enabled with the [`ENABLE_PRO_PRICING`](
+  #enable_pro_pricing
+) configuration setting.
+
+With the feature enabled, users will be able to submit their bank card details
+and subscribe to a recurring subscription plan which will give them access to
+all the professional features.
+
+## Payment gateway
+
+For a payment gateway Alaveteli Professional uses [Stripe](https://stripe.com).
+It is widely available around the world and is easy to configure and work with.
+
+The main advantage of Stripe is that all payment information is entered and sent
+directly to Stripe. They never gets transmitted to or via your Alaveteli
+installation so the security and data protection of those details are not a
+concern for you.
+
+### Creating an account
+
+Signing up for an account with Stripe is quick and easy. Initially your account
+will be a limited test account.
+
+Before you can take payments you will need to activate your account. You will
+need to provide legal information on you and/or your company as well as your
+bank details in order receive payments.
+
+Once approved by Stripe you'll then be able to receive payments and Stripe will
+transfer the money directly into your bank account.
+
+## Stripe configuration
+
+Stripe provides a secure and robust Application Programming Interface or API
+which Alaveteli Professional uses to communicate with Stripe. In order to use
+this we require some configuration settings from your Stripe dashboard.
+
+Follow the instructions below to set this up for your Alaveteli installation.
+
+* [API keys](#api-keys)
+* [Webhook endpoint](#webhook-endpoint)
+* [Setting up the subscription product](#setting-up-the-subscription-product)
+* [Creating coupons](#creating-coupons)
+* [Using a namespace](#using-a-namespace)
+
+### API keys
+
+For Alaveteli Professional to communicate securly with Stripe you'll need to
+generate a pair of [standard API keys](https://dashboard.stripe.com/apikeys) and
+add the [`STRIPE_PUBLISHABLE_KEY`](#stripe_publishable_key) and
+[`STRIPE_SECRET_KEY`](#stripe_secret_key) settings.
+
+### Webhook endpoint
+
+Stripe uses [webhooks](https://dashboard.stripe.com/webhooks) to inform the
+Alaveteli installation of important events, such as when a users bank card can't
+be charged or when a subscription has been cancelled.
+
+You should add a endpoint pointing to `/pro/subscriptions/stripe-webhook` for
+your Alaveteli installation.
+
+The event types which should be subscribed to are:
+* `customer.subscription.deleted`
+* `invoice.payment_succeeded`
+* `invoice.payment_failed`
+* `customer.subscription.updated`
+
+Finally you'll set the [`STRIPE_WEBHOOK_SECRET`](#stripe_webhook_secret) to the
+a signing secret found in the Stripe dashboard.
+
+### Setting up the subscription product
+
+A subscription product in Stripe needs to be [created](
+  https://dashboard.stripe.com/subscriptions/products/create
+). We recommend "Pro", but this can have any name you wish.
```

Comment:
Worth adding something like… "but bear in mind that this name will be displayed to users"

e.g. During signup

<img width="805" alt="Screen Shot 2019-07-18 at 15 19 36" src="https://user-images.githubusercontent.com/27760/61465344-cd292300-a96f-11e9-8f05-3305967c50df.png">

On the user's subscription page

<img width="306" alt="Screen Shot 2019-07-18 at 15 24 18" src="https://user-images.githubusercontent.com/27760/61465548-2abd6f80-a970-11e9-8751-9e39f17b86f8.png">

And I think we use it in the charge description as well, although the Stripe invoice seems to use the Product name instead.

Reply:
Yep good call added.